build.gradle

```diff
@@ -280,14 +280,26 @@ task dbWait(dependsOn: 'compileJava', type: JavaExec) {
 }
 dbWait.mustRunAfter('dockerStartDB')
 
+// Create the task that runs the initialization command. Run e.g. with:
+//   ./gradlew dbInit -Pargs="--adminOrgName <adminOrgName> --adminPosName <adminPosName>
+//   --adminFirstName <adminFirstName> --adminLastName <adminLastName> --adminDomainName <adminDomainName>"
+// To see all options, run just:
+//   ./gradlew dbInit
+// which will print the help.
 task dbInit(dependsOn: 'compileJava', type: JavaExec) {
 	group = "database runtime"
 	description = "Runs the ANET database initialization command."
 	classpath = sourceSets.main.runtimeClasspath
 	environment(run.environment)
 	mainClass = mainClassName
-	args = ["init", anetConfig]
-	standardInput = System.in
+	def cmdline = ["init"]
+	if (project.hasProperty("args")) {
+		cmdline.addAll(project.args.split("\\s+"))
```

Comment:
This won't allow spaces in the argument values; something like:
```
               cmdline.addAll(project.args.split("(?<!\\\\)\\s+").collect { a -> a.replace("\\ ", " ")})
```
would do that.